deep/settings.py

```diff
@@ -83,7 +83,7 @@
     'user_resource',
     'tabular',
     'commons',
-
+    'notification',
```

Comment:
Add a new line after `notification`. 
right now the block for the install_app is as
```
# Django apps
# lib apps
# Deep apps
# Misc apps
```